components/raftstore/src/store/peer.rs

```diff
@@ -744,6 +744,7 @@ impl Peer {
             if let LeaseState::Valid = state {
                 let mut resp = eraftpb::Message::default();
                 resp.set_msg_type(MessageType::MsgReadIndexResp);
+                resp.term = self.term();
```

Comment:
Better check if all required fields are set. I can see there are still places that construct raft messages by ourselves, need to be checked too.

Reply:
Another raft message constructed by ourselves is `transfer_leader` but it is local msg for raft-rs.In TiKV we send it for checking transfer leader condition and do not use term field so it's ok not to set it.